commitlint.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: ["@commitlint/config-conventional"],
+  formatter: "@commitlint/format",
+
+  parserPreset: {
+    parserOpts: {
+      issuePrefixes: ["ABC-", "#"],
```

Comment:
We will most likely only refer to our issues with "#", e.g. #12

```suggestion
      issuePrefixes: ["#"],
```